roles/dns-slave/files/named.conf.local

```diff
@@ -0,0 +1,18 @@
+zone "sown.org.uk" {
+	type slave;
+	check-names ignore;
+	masters { 10.5.0.239; 2001:630:d0:f700::239; };
+	file "/var/cache/bind/db.sown.org.uk";
+};
+
+zone "5.10.in-addr.arpa" {
+        type slave;
+        masters { 10.5.0.239; 2001:630:d0:f700::239; };
```

Comment:
We should configure this

Reply:
Hmm, I initially started writing the playbook planning to template all this, then concluded we end up worthlessly converting between DSLs, and removing the flexibility at the same time, particularly considering the config is identical betwen gw/gw2.

Reply:
Oh, I suppose we could always break them out separately in the config to avoid repetition though

Reply:
Done - we could always use catalogue zones when we redo it I suppose, but this will do for now.